src/components/Chat/ChatContent/Message/MessageContent.tsx

```diff
@@ -495,16 +506,18 @@ const EditViewButtons = React.memo(
             </button>
           )}
 
-          <button
-            className={`btn relative mr-2 ${
-              sticky ? 'btn-neutral' : 'btn-primary'
-            }`}
-            onClick={handleSave}
-          >
-            <div className='flex items-center justify-center gap-2'>
-              {t('save')}
-            </div>
-          </button>
+          {advancedMode && (
```

Comment:
I think the Save button should be available in Simple Mode. 
This is because it is useful to fix or adjust language spelling without having to resend the text to the AI if the generated answer was acceptable despite the spelling issues.

Reply:
The simple mode is designed for beginners who may not be familiar with the technicalities behind the scenes. Including both "save" and "save and commit" buttons could cause confusion among some users. Adopting a save-commit only approach aligns with the behavior of Official chat.openai.com, making it more intuitive for users. 

if you require a save button without committing changes, you can always switch to advanced mode anyway.

Does this argument seem convincing to you?

Reply:
I think peng1999 makes sense.

Reply:
I'll need to find time to review this PR.

Reply:
> The simple mode is designed for beginners who may not be familiar with the technicalities behind the scenes. Including both "save" and "save and commit" buttons could cause confusion among some users. Adopting a save-commit only approach aligns with the behavior of Official chat.openai.com, making it more intuitive for users.
> 
> if you require a save button without committing changes, you can always switch to advanced mode anyway.
> 
> Does this argument seem convincing to you?

It makes sense, but since this project was in the way to not follow exactly the UX behavior of original chat.openai.com. Renaming the button "save and submit" to just "submit", then to have "save" and "submit" button can be a workaround as I mentioned in #182 . If in simple mode, we just leave user edit thing just to make them confirm to erase the rest of the messages coming next, then enable advanced mode can be "overheard" to just maybe save a spelling correction without lost the AI current answer which is maybe accurate despite the spelling issue. 

If simple mode have to follow chat.openai.com UX behavior, then user have to not also be able to edit assistant message and make assistant submit message to himself. In advanced mode, there can be use cases for that, but not in simple mode if, just as I said, to adjust the message and save to make the answer more as expected for the user.